src/app/components/chat-box/chat-box.component.ts

```diff
@@ -67,12 +69,21 @@ export class ChatBoxComponent {
       //  this.route.navigate([''])
     }
     this.UserSelected = 'Test';
+    this.chatservice.getSocketData('groupCreated').subscribe((res) => {
```

Comment:
Push newly created group to the existed groups

Reply:
done